datalad/support/gitrepo.py

```diff
@@ -1977,16 +1977,18 @@ def get_merge_base(self, commitishes):
             commitishes = commitishes + [self.get_active_branch()]
 
         try:
-            bases = self.repo.merge_base(*commitishes)
-        except GitCommandError as exc:
-            if "fatal: Not a valid object name" in str(exc):
-                return None
-            raise
+            bases = self.call_git_oneline(['merge-base'] + commitishes)
```

Comment:
`bases` made sense given the gitpython return value, but I think for clarity (because we don't support `--all`) the rewrite should use `base`.

Reply:
Will do.